### ISSUE TITLE
Change code block style in the docs

### ISSUE THREAD

**Issue by kaedroho** [wagtail/wagtail#6432]
The colours in our existing code blocks fail WCAG AA on contrast: https://webaim.org/resources/contrastchecker/?fcolor=408090&bcolor=EEFFCC

See an example here: https://docs.wagtail.io/en/stable/advanced_topics/performance.html#cache

It looks like ``sphinx-rtd-theme`` uses a different style for their own docs: https://sphinx-rtd-theme.readthedocs.io/en/latest/demo/demo.html#code-blocks so maybe we should switch to that.
